src/types.ts

```diff
@@ -0,0 +1,64 @@
+import CID from 'cids'
+import PeerId from 'peer-id'
```

Comment:
i think theres a PeerId type being defined somewhere else in these changes.

Reply:
it is imported in couple of files where they are used